walkthroughs/3-low-code-api-development/walkthrough.adoc

```diff
@@ -107,7 +116,7 @@ Verify that you followed each step in the procedure above.  If you are still hav
 
 === Creating a POST Resource Path
 
-A *Path* represents an API endpoint/operation and the associated parameters required to invoke it.
+Follow these steps to create a *Path*. A *Path* represents an API endpoint and the associated parameters required to invoke it.
```

Comment:
I like keeping the info about the path above the "Follow these steps..." section. That way if we add more context, additional info etc, it's all in the same location. After the title/heading and before the steps.

Reply:
Going to merge this for now so we can see where things are at going e2e. This comment is for a "nice-to-have"